server/services/userService.ts

```diff
@@ -1,18 +1,16 @@
 import convertToTitleCase from '../utils/utils'
-import type HmppsAuthClient from '../data/hmppsAuthClient'
-
-type User = Express.User
+import ManageUsersApiClient from '../data/manageUsersApiClient'
 
 interface UserDetails {
   name: string
   displayName: string
 }
 
 export default class UserService {
-  constructor(private readonly hmppsAuthClient: HmppsAuthClient) {}
+  constructor(private readonly manageUsersApiClient: ManageUsersApiClient) {}
 
-  async getUser(userDetails: User): Promise<UserDetails> {
-    const user = await this.hmppsAuthClient.getUser(userDetails)
-    return { ...user, displayName: convertToTitleCase(user.name as string) }
+  async getUser(userDetails: Express.User): Promise<UserDetails> {
+    const user = await this.manageUsersApiClient.getUser(userDetails)
+    return { name: user.name, displayName: convertToTitleCase(user.name as string) }
```

Comment:
```suggestion
    return { ...user, name: user.name, displayName: convertToTitleCase(user.name as string) }
```

Not sure if you need to destructure the user here as well?

Reply:
good spot, the service actually only uses the diaplayName from this object.. but makes sense to destructure here so other attributes are also available if needed in the future.

Fixed and pushed